perf/perf_sdt_probe.py

```diff
@@ -34,14 +34,14 @@ class PerfSDT(Test):
     """
 
     def run_cmd(self, cmd):
-        if process.system(cmd, ignore_status=True, sudo=True, shell=True):
+        if process.system(cmd, sudo=True, shell=True):
```

Comment:
any reason to `ignore_status=True` to removig this

Reply:
If any perf sdt event fails, the test should fail. If we add `ignore_status=True` it will ignore the failure.